packages/vite/src/node/plugins/assetImportMetaUrl.ts

```diff
@@ -92,8 +85,8 @@ export function assetImportMetaUrlPlugin(config: ResolvedConfig): Plugin {
             map: config.build.sourcemap ? s.generateMap({ hires: true }) : null
           }
         }
+        return null
```

Comment:
Why was this `return null` moved up here? Is it needed at all? We could let the function finish without returning, no?

Reply:
inadvertently changed 🙈

IIUC, no return, and return null had the same behavior.